README.md

```diff
@@ -1,5 +1,130 @@
-# Hackathon
+# Global Legal Hackathon
 
 <br>
 
+### Project Contacts
+
+Matthew Schneider
+    [INSERT]
+
+Jordan L. Fischer, Esq.
+    Teaching Professor of Law
+    Thomas R. Kline School of Law at Drexel University
+    [jlf324@drexel.edu](jlf324@drexel.edu)
+
+Cameron Bale
+    [INSERT]
+
+### Location-Based Data Privacy Solution
+
+This Project addresses the trade-off between the accuracy of location data with privacy and legal privacy protections.
+
+Prior research (de Montjoye et al. 2013) shows that four spatio-temporal points are enough to uniquely identify 95% of individuals. This means that the assortment of COVID-19 location-tracking apps are destined to identify some individuals. Privacy concerns are further exacerbated when geospatial points can be traced to home addresses which are linked to public tax and real estate sale records.
+
+### Project Goals
+
+For the purpose of this FT Global Legal Hackathon, this project addresses the following core components: 
+
+* Statistical process to reduce the identifiability of individuals while maintaining most of the usefulness of the data
+
+* Differentially private histogram approach for geospatial data which provides a theoretical guarantee to individual privacy.
```

Comment:
This is more of a future solution, but I'm fine to leave it here if needed

Reply:
@mjs624 agreed so I moved it to the future work section!